pyccel/epyccel.py

```diff
@@ -277,8 +284,10 @@ def epyccel( python_function_or_module, **kwargs ):
 
     Parameters
     ----------
-    python_function_or_module : function | module
+    python_function_or_module : function | module | str
         Python function or module to be accelerated.
+        If a string is passed then it is assumed to be the code from a module which
+        should be accelerated..
```

Comment:
Probably worth mentioning: this code must include any necessary `import` statements.

Reply:
36894f12a29a880b655b65aff24affd404b7abb2